runtime/jvmti/jvmtiMemory.c

```diff
@@ -80,4 +81,20 @@ jvmtiDeallocate(jvmtiEnv* env,
 	TRACE_JVMTI_RETURN(jvmtiDeallocate);
 }
 
+jvmtiError JNICALL 
+jvmtiSetHeapSamplingInterval(jvmtiEnv *env, 
+	jint samplingInterval)
+{
+	jvmtiError rc = JVMTI_ERROR_NONE;
+	
+	Trc_JVMTI_jvmtiSetHeapSamplingInterval_Entry(env, samplingInterval);
+	
+	ENSURE_PHASE_ONLOAD_OR_LIVE(env);
+	ENSURE_CAPABILITY(env, can_generate_sampled_object_alloc_events);
+	ENSURE_NON_NEGATIVE(samplingInterval);
+
+	assert(!"jvmtiSetHeapSamplingInterval unimplemented yet"); /* https://github.com/eclipse/openj9/issues/3436 */
```

Comment:
No - just return an error code for now.

Reply:
Returning an error `JVMTI_ERROR_UNSUPPORTED_VERSION` now, please have another look.